qds_sdk/cluster.py

```diff
@@ -362,6 +362,33 @@ def clone(cls, cluster_id_label, cluster_info):
         return conn.post(cls.element_path(cluster_id_label) + '/clone', data=cluster_info)
 
     @classmethod
+    def _parse_cluster_manage_command(cls, args):
+      """
+      Parse command line arguments for cluster manage commands.
+      """
+      argparser = ArgumentParser(prog="cluster_manage_command")
+
+      group = argparser.add_mutually_exclusive_group(required=True)
+
+      group.add_argument("--id", dest="cluster_id",
+                           help="execute on cluster with this id")
+
+      group.add_argument("--label", dest="label",
+                           help="execute on cluster with this label")
+
+      argparser.add_argument("--parameters",
+                 help="optional parameters to the manage command")
+
+      argparser.add_argument("--private_dns",
+                 help="the private_dns of the machine to be updated/removed")
```

Comment:
This is valid for replace only ? If yes, then put it under an if condition. Maybe it time to have a class hierarchy on the side. *ClusterApiCmdLine* Too much repetition. For e.g. check snapshot/restore_point. Anyway, cleanup for another day.